recipes/flaightidl/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "flaightidl" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: f19969361b2167fc349a8ddef97833006083ebc7a39eef69f4968d6c987f7f89
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - protobuf >=3.5.0,<4.0.0
+    - python
+  run:
+    - protobuf >=3.5.0,<4.0.0
+    - python
```

Need a python version for noarch:
```suggestion
    - python >=3.6
  run:
    - protobuf >=3.5.0,<4.0.0
    - python >=3.6
```